pkg/networkservice/common/monitor/passthrough_test.go

```diff
@@ -249,6 +249,7 @@ func (m *MonitorPassThroughSuite) TestServerUpdate() {
 	expectedConn := endpointConn.Clone()
 	expectedConn.GetPath().Index = m.conn.GetPath().GetIndex()
 	expectedConn.Id = expectedConn.GetCurrentPathSegment().GetId()
+	expectedConn.State = networkservice.State_DOWN
```

Comment:
Why do we need to change this?

Reply:
We modified `begin` to change the connection state to `UP`.
In this test we didn't do `connection.Clone()` when calling Request, so we ended up with a modified Connection object, which I tried to fix by manually reverting the connection to expected state.

I have now changed it to use `Clone` instead of manually modifying the object.